City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# UHF-3821: 'Show website link' checkbox for TPR Unit edit form

How to test:

1. Set up KASKO environment
2. Checkout this branch: `git checkout UHF-3821_tpr_unit_website_link`
3. Checkout equivalent branch of the `helfi_tpr` module: `composer require drupal/helfi_tpr:dev-UHF-3821_tpr_unit_website_link`
4. Run `drush updb` to update the changes from the TPR module
5. Run `drush cim` to import the config changes from this branch
6. Edit a TPR Unit with "Website link" field and see how the new "Show website link" selection affects the visibility of the link field when viewing the entity

Also see PR's:

- https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/91
- https://github.com/City-of-Helsinki/drupal-helfi-platform-config/pull/243
